gooseai/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/stability-ai/api-interfaces/gooseai
 
-go 1.18
+go 1.17
```

Comment:
We want to keep this at `go 1.18`

Reply:
yeah I have no idea why that changed and it freaked me out a bit when I first saw it. might need to force a version in one of the CI actions